setup.py

```diff
@@ -16,6 +16,7 @@
       packages=['pipeline_tools'],
       install_requires=[
           'cromwell-tools',
```

Comment:
I think we can remove the cromwell-tools as a dependency of `pipeline-tools` for now? (Since we only used cromwell-tools in `cromwell_tools.generate_auth_header_from_key_file(json_credentials)`)